osu.Game/Screens/Play/HUD/HoldForMenuButton.cs

```diff
@@ -60,6 +60,18 @@ public HoldForMenuButton()
             AutoSizeAxes = Axes.Both;
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            var activationDelay = config.GetBindable<int>(OsuSetting.UIHoldActivationDelay).GetBoundCopy();
```

Comment:
This is going to be GC'd and references are going to be lost.

Reply:
in two ways. spot them both (hint: check *any* other usage of config manager)